pkg/cmd/stack/translate.go

```diff
@@ -138,7 +138,6 @@ func translateDeployment(svcName string, s *model.Stack) *appsv1.Deployment {
 							Resources:       translateResources(svc),
 							WorkingDir:      svc.Workdir,
 							ReadinessProbe:  healthcheckProbe,
-							LivenessProbe:   healthcheckProbe,
```

Comment:
Is this only for healthchecks that are added from compose files? I see how this will help debug issues that are no longer visible due to restarts but we may still have problems. In this new scenario the pod will be deployed but kube will not route traffic to it. Do you foresee that as being a similar issue that we should handle as well?

Reply:
Yes, it'll only affect compose and `okteto stack` files

Reply:
`pod will be deployed but kube will not route traffic to it` -> it will if the readiness probe is passing. That is exactly what a user wants when he defines healthchecks at the compose file. the liveness was just introducing noise and making things harder to debug